From aa2c9613d704597326db68d00999951326ded14e Mon Sep 17 00:00:00 2001
From: Srujana Challa <schalla@marvell.com>
Date: Wed, 17 Jun 2020 15:05:42 +0530
Subject: [PATCH 0518/1921] crypto: marvell: fix Makefile for octeontx2

Fixes build issue for octeontx2 cpt driver, which is
being compiled with CONFIG_CRYPTO_DEV_OCTEONTX_CPT config
option, but it should be CONFIG_CRYPTO_DEV_OCTEONTX2_CPT.
Fixes: 9741f4ecc388
("crypto: marvell: enable OcteonTX2 cpt options for build")

Change-Id: I30fe8e968617b3ad6c795c323f56a1152337c765
Signed-off-by: Srujana Challa <schalla@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/30097
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/crypto/marvell/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/marvell/Makefile b/drivers/crypto/marvell/Makefile
index 7c18a53153b1..74894c4e127b 100644
--- a/drivers/crypto/marvell/Makefile
+++ b/drivers/crypto/marvell/Makefile
@@ -2,4 +2,4 @@
 
 obj-$(CONFIG_CRYPTO_DEV_MARVELL_CESA) += cesa/
 obj-$(CONFIG_CRYPTO_DEV_OCTEONTX_CPT) += octeontx/
-obj-$(CONFIG_CRYPTO_DEV_OCTEONTX_CPT) += octeontx2/
+obj-$(CONFIG_CRYPTO_DEV_OCTEONTX2_CPT) += octeontx2/
-- 
2.31.1

